tuftool/src/create.rs

```diff
@@ -77,202 +63,49 @@ pub(crate) struct CreateArgs {
 
 impl CreateArgs {
     pub(crate) fn run(&self) -> Result<()> {
-        if let Some(jobs) = self.jobs {
-            rayon::ThreadPoolBuilder::new()
```

Comment:
rayon didn't seem to actually be used in refresh.rs, so I understand that removal, but I think it's used here in create to parallelize building targets, and I think we're losing that with the move to the repo editor... right?  Does it take a lot longer to create repos?  I wonder if we should include that as an improvement to repo editor in another commit for this PR?

Reply:
Per offline discussion, tough shouldn't own the parallelism, but tuftool could/should.  If we can add `Target`s individually to the editor, rather than just paths, then we can build `Target`s in parallel in tuftool and add them to the editor as they're ready.  I don't think it has to be done here, but it does feel unfortunate to lose the parallelism with the change...

Reply:
We should probably add Windows to the build matrix. Easy to do, seems relevant with respect to symlinks, unless they 'just work' the same over there.